Sources/swift-test/main.swift

```diff
@@ -21,11 +21,14 @@ do {
         usage()
     case .Run(let xctestArg):
         let dir = try directories()
+
+        //FIXME find a reliable name to detect the name of the root test Package
+        let testPackageName = dir.root.basename
```

Comment:
I'm all ears on how to pull the root package's name. We can always reparse the manifest, but for now it feels like an overkill. @mxcl?